### PR TITLE
Add -save-temps option to README

### PR DESCRIPTION
**Pull request by HorstBaerbel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,19 +18,21 @@ Objdumped disassembly should be generated with lines info:
 
     objdump -d compiled.bin -l > disassembly.S
 
-Next, when you generated disassembly, you should tell to extension, where to
-search disassembly. By default it is searched right near the source file, but
-with `.S` extension:
+When you don't want to specify the output file manually (or use e.g. CMake or build scripts) and compile with GCC, you can use the [-save-temps option](https://gcc.gnu.org/onlinedocs/gcc/Developer-Options.html). This will put a `source.s` file into the output directory. Compile with:
```
Comment (dseight) [inline README.md:21]: Maybe it's better to give a link directly to the `-save-temps` option? 
```suggestion
When you don't want to specify the output file manually (or use e.g. CMake or build scripts) and compile with GCC, you can use the [-save-temps option](https://gcc.gnu.org/onlinedocs/gcc/Developer-Options.html#index-save-temps). This will put a `source.s` file into the output directory. Compile with:
```

File: README.md [hunk 2]
```diff
@@ -2,16 +2,13 @@
 
 Shows disassembly to source relations and dims unused source lines.
 
-This extension is based on [Compiler Explorer](https://github.com/mattgodbolt/compiler-explorer)
-(especially on AsmParser class from it).
+This extension is based on [Compiler Explorer](https://github.com/mattgodbolt/compiler-explorer) (especially on AsmParser class from it).
```
Comment (dseight) [inline README.md:5]: Also it is always better to leave lines unrelated to change in their original form.
```suggestion
This extension is based on [Compiler Explorer](https://github.com/mattgodbolt/compiler-explorer)
(especially on AsmParser class from it).
```

File: README.md [hunk 3]
```diff
@@ -21,19 +18,21 @@ Objdumped disassembly should be generated with lines info:
...
 "disasexpl.associations": {
     "**/*.c": "${fileDirname}/${fileBasenameNoExtension}.S",
     "**/*.cpp": "${fileDirname}/${fileBasenameNoExtension}.S"
 }
 ```
+Make sure to adjust the disassembly extension to `.s` when using the "-save-temps" option.
 
-Now, when all is done, simply select `F1 > Disassembly Explorer: Show`.
-Disassembly will be opened on the right side of your source.
+Now, when all is done, simply select `F1 > Disassembly Explorer: Show`. Disassembly will be opened on the right side of your source.
```
Comment (dseight) [inline README.md:35]: Same as previous.
```suggestion
Now, when all is done, simply select `F1 > Disassembly Explorer: Show`.
Disassembly will be opened on the right side of your source.
```

**Review by dseight — APPROVED** [review]
Thank you for correcting my terrible English! :) Can you please update PR with suggested changes? Sorry for such a late response - I have just forgot to submit a review.

File: README.md (hunk 1 above)
Comment (HorstBaerbel) [reply README.md:21]: Even better!

File: README.md (hunk 3 above)
Comment (HorstBaerbel) [reply README.md:35]: I wouldn't put so many unneeded line breaks here. If you watch the file on another device you want to make the text adapts to the screen as much as possible...

Comment (dseight) [reply README.md:35]: Line breaks are present here to fit all the text in 80 symbols. That makes it possible to read it in _any_ editor, even without word wrap support (or with word wrap disabled).

On a device one probably will read rendered markdown instead of this raw text. And when it's rendered, this line breaks will not be visible.

Comment (HorstBaerbel) [reply README.md:35]: I personally don't see a use case for the 80 characters limit, but maybe that's just me. But anyway. Edit the pull request as needed. It's your project.
